src/app-logic/url-handling.js

```diff
@@ -365,6 +368,7 @@ export function stateFromLocation(
     selectedTab: toValidTabSlug(pathParts[selectedTabPathPart]) || 'calltree',
     pathInZipFile: query.file || null,
     profileName: query.profileName,
+    showTabOnly: query.showTabOnly ? Number(query.showTabOnly) : null,
```

Comment:
maybe check that the result of the cast to a number is correct (`Number.isInteger` could help here)